### PR TITLE
V2 schema for phone auth

### PR DESCRIPTION
**Pull request by jrbuhl93**
Added a split to ApolloClient links so that we use `userRequestAuthCode` and `userLogin` from the new graphql api.

### CODE REVIEW

File: app/screens/phone-auth-screen/phone-auth.tsx [hunk 1]
```diff
@@ -93,15 +98,14 @@ type WelcomePhoneInputScreenProps = {
...
   navigation,
 }: WelcomePhoneInputScreenProps) => {
-  const [requestPhoneCode, { loading }] = useMutation(REQUEST_PHONE_CODE, {
+  const [requestPhoneCode, { loading }] = useMutation(REQUEST_AUTH_CODE, {
     fetchPolicy: "no-cache",
   })
 
   const inputRef = useRef<PhoneInput | null>()
 
   const send = async () => {
```
Comment (mfferreira) [inline app/screens/phone-auth-screen/phone-auth.tsx:107]: What do you think about including `useCallback` hook here? Can be part of a separate code cleanup PR.

Comment (jrbuhl93) [reply app/screens/phone-auth-screen/phone-auth.tsx:107]: Is it not already? https://github.com/apollographql/react-apollo/issues/3260

Comment (mfferreira) [reply app/screens/phone-auth-screen/phone-auth.tsx:107]: I'm referring to the `send` function.

```suggestion
  const send = useCallback(async () => {
```

Comment (jrbuhl93) [reply app/screens/phone-auth-screen/phone-auth.tsx:107]: Oh, then yes. In general we aren't using enough useCallbacks throughout the app. I can add it now.

**Review by mfferreira — APPROVED** [review]

**Review by samerbuna — APPROVED** [review]

**Pull request merged**